Doc/faq/general.rst

```diff
@@ -248,7 +248,7 @@ Are there any published articles about Python that I can reference?
 
 It's probably best to cite your favorite book about Python.
 
-The very first article about Python was written in 1991 and is now quite
+The `very first article <https://ir.cwi.nl/pub/18204>`_ about Python was written in 1991 and is now quite
 outdated.
```

Comment:
Let's stick to max 80 chars:

https://devguide.python.org/documentation/style-guide/#use-of-whitespace

```suggestion
The `very first article <https://ir.cwi.nl/pub/18204>`_ about Python was
written in 1991 and is now quite outdated.
```

A couple of tips:

* You can click "Commit suggestion" to directly apply this to the PR
* No need to click the "Update branch" button every time, it just creates unnecessary builds

Reply:
okay!!
Thank you so much for helping me out.
This is my contribution that's why you are having so much trouble working with me.
I apologize for that.

Reply:
No need to apologise, thank you for the updates!